README.md

```diff
@@ -0,0 +1,138 @@
+
+# ForumAI
+
+ForumAI is an online platform which enables AI assistance in structured academic forums, facillutating discussions between students and faculty.
+
+
+## Features
+
+- 24/7 AI assistance to help answer questions about course material, assigments and general logistics
+- Discussions boards and Q&A forums for interaction between students, instructors and teaching assistants
+- Anonoymous posting and messaging for safe and inclusive discussions
+
+## Installation
+
+Clone the project
+
+```bash
+git clone https://github.com/ubclaunchpad/forum.git
+```
+
+### Frontend
+
+#### 1. Install Node.js
+- Download and install Node.js from [nodejs.org](https://nodejs.org/en)

```

Comment:
Mention to install the latest node version (e.g. Download and install the latest Node.js version from [nodejs.org](https://nodejs.org/en))